mkosi.profiles/desktop/mkosi.conf.d/fedora/mkosi.conf

```diff
@@ -11,5 +11,4 @@ Packages=
         mesa-dri-drivers
         mesa-vulkan-drivers
         NetworkManager
-        microcode_ctl
-        amd-ucode-firmware
+        upower
```

Comment:
Didn't we just switch to `tuned` in Fedora 41?

Reply:
Uh that's a good point, I didn't see `power-profiles-daemon` on any of my Fedora systems, which is why I moved it under arch. I'll double check on tuned. btw, at least on Fedora tuned won't start unless `/etc/tuned` is present, so we might need to add that too to tmpfiles (or fix it upstream).

Reply:
Ok, looks like on Fedora we should use `tuned-ppd` instead of `power-profiles-daemon`: https://fedoraproject.org/wiki/Changes/TunedAsTheDefaultPowerProfileManagementDaemon